lib/core/src/sdk.rs

```diff
@@ -453,6 +453,15 @@ impl LiquidSdk {
     /// Should only be called once per instance.
     pub async fn start(self: &Arc<LiquidSdk>) -> SdkResult<()> {
         let mut is_started = self.is_started.write().await;
+
+        #[cfg(all(target_family = "wasm", target_os = "unknown"))]
+        if let Err(e) = self
+            .persister
+            .restore_from_indexed_db_backup(&self.onchain_wallet.fingerprint()?)
+            .await
+        {
+            error!("Failed to restore from IndexedDB backup: {:?}", e);
+        }
```

Comment:
Since this code is specific to wasm better I think move it to the wasm crate. We already initializing the in memory persister there so it seems naturally belongs there.